src/components/VHeader/VHeader.vue

```diff
@@ -1,22 +1,64 @@
+        />
+      </svg>
     </NuxtLink>
 
+    <VSearchBar
+      v-show="!isHomeRoute"
+      v-model.trim="searchTerm"
+      class="mx-auto md:ms-0 md:me-auto lg:w-1/2 2xl:w-1/3"
+      :class="{
+        'order-4 w-full md:order-none md:w-auto': !isHeaderScrolled,
+        'w-2/3': isHeaderScrolled,
+      }"
```

Comment:
nitpick: The array syntax fit better here.
```suggestion
      :class="[
        isHeaderScrolled ? 'w-2/3' : 'order-4 w-full md:order-none md:w-auto',
      ]"
```